CMakeLists.txt

```diff
@@ -49,6 +49,7 @@ macro(build_spdlog)
   list(APPEND extra_cmake_args "-DSPDLOG_BUILD_TESTS=OFF")
   list(APPEND extra_cmake_args "-DSPDLOG_BUILD_EXAMPLE=OFF")
   if(NOT WIN32)
+    list(APPEND extra_cmake_args "-DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS} -Wno-sign-conversion")
```

Comment:
Can you add a TODO comment here with a link the upstream `fmt` ticket?

Reply:
done in https://github.com/ros2/spdlog_vendor/pull/7/commits/aaf27f3bfd9075284424f6bc3916b5dade5bdeff